src/cheri/insns/acperm_32bit.adoc

```diff
@@ -16,8 +16,6 @@ include::new_encoding_note.adoc[]
 Synopsis::
 {ACPERM_DESC}
```

Comment:
I think we should probably also change the macro names from ACPERM to CLRPERM or similar to show that it's different now.